input/data/feed_management/bale_size.json

```
@@ -0,0 +1,3 @@
+{
+  "bale_diameter": 1.5
```

```suggestion
  "hay_bale_diameter": 1.5
```
I recommend making this more specific, because it applies only to hay and not to baleage